quipucords/api/scan/model.py

```diff
@@ -65,7 +84,10 @@ class ScanOptions(models.Model):
     JBOSS_BRMS_EXT = 'jboss_brms_ext'
 
     max_concurrency = models.PositiveIntegerField(default=50)
-    disable_optional_products = models.TextField(null=True)
+    disable_optional_products = \
```

Comment:
confused from the last PR I thought it was "disabled" now I just see "disable" and its not consistent with "enabled"